components/PoweredBy/index.js

```diff
@@ -0,0 +1,29 @@
+var React = require('react'),
+    bem = require('../BemHelper')('as-powered-by'),
+    logo = require('url?limit=10000!./algolia_logo.png');
+
+require('style?prepend!raw!./index.css');
```

Comment:
So it's:
- getting the raw css content from the file
- creating a `<style>content</style>`
- prepending it to all other stylesheets

Prepend to allow users to override our styles (example: slider).